src/modules/lua_mtev_crypto.c

```diff
@@ -84,7 +90,7 @@ mtev_lua_crypto_x509_index_func(lua_State *L) {
   cert = *((X509 **)udata);
   if(!strcmp(k, "signature_algorithm")) {
     int nid;
-    nid = OBJ_obj2nid(cert->sig_alg->algorithm);
+    nid = X509_get_signature_nid(cert);
```

Comment:
What does this change here do?

Reply:
It doesn't change anything, but cert is opaque in openssl 1.1